daffodil-core/src/main/scala/org/apache/daffodil/dsom/AnnotatedSchemaComponent.scala

```diff
@@ -1,485 +1,482 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
```

Comment:
This entire file shows as a diff. Did you change the line endings perhaps? We try to always use unix lined endings. Make sure you IDE isn't changing things. It's not clear what changes were actually made to this and other files.